@navikt/core/react/src/form/form-summary/FormSummaryEditLink.tsx

```diff
@@ -24,17 +16,23 @@ export const FormSummaryEditLink: OverridableComponent<
   HTMLAnchorElement
 > = forwardRef(({ children, className, as = "a", ...rest }, ref) => {
   const { cn } = useRenameCSS();
+
   const translate = useI18n("FormSummary");
 
   return (
-    <Link
-      ref={ref}
-      as={as}
-      {...rest}
-      className={cn("navds-form-summary__edit", className)}
+    <CompositionWarning.Forbidden
+      name="FormSummary.Header"
+      message="<FormSummary.EditLink> should not be placed in <FormSummary.Header>. See: https://aksel.nav.no/komponenter/core/formsummary"
```

Comment:
I suggest emphasizing that it hasn't always been like this:

```suggestion
      message="<FormSummary.EditLink> should not be placed in <FormSummary.Header> anymore. See https://aksel.nav.no/komponenter/core/formsummary"
```